### ISSUE TITLE
Bug report: Shot count and track assets not showing in end screen.

### ISSUE THREAD

**Issue by Aphex123** [unitedoperations/UnitedOperationsFramework#109]
#### Description of issue

On end screen; end condition messages show properly, but shot count and track assets does not show. Just blank below end condition message.

ON version 1.0.5.
Using all mods used on UO server.
#### Mission File(s)

https://www.dropbox.com/s/dgcudy1sb4fwjts/TVT42_Pillar_of_Ohmen_V2.ruha.zip?dl=0

#### Steps to reproduce

1. End mission after shots have been shot and assets have been destroyed.
2. Observe

#### Expected behaviour

End screen would show end condition message with shots fired and assets destroyed below.

#### Video and Screenshots

https://steamcommunity.com/sharedfiles/filedetails/?id=1723282767

This was with numerous shots fired and 1x hmmv destroyed.

#### Log

https://pastebin.com/3HG3uKfD

**Comment by thecogdoctor** [comment]
Please include logs. This is important for the debug process.
Please also include information on what version of the framework your using and any other mods you are loading.

Recommend mods are CBA, UO ACRE2, ace, and the version of the framework you are running.

**Comment by thecogdoctor** [comment]
@ChrSacher In a separate PR after #108 let's work on addressing this issue and the other feature requests in a separate branch.

**Comment by Aphex123** [comment]
Updated

**Comment by ChrSacher** [comment]
Adressed in #113

**Comment by thecogdoctor** [comment]
Fixed in #113.
